### PR TITLE
Roundup 325 typo fixes

### PR DESCRIPTION
**Pull request by Gama11**

### CODE REVIEW

**Comment by skial** [comment]
Not as bad as I thought it would be :+1:

File: src/roundups/325.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ language”_.
 located in the Netherlands _“mainly related to TV programs that are broadcast on 
 the channels of RTL Netherlands”_.
 
-As usual there has been a bunch of HaxeLib releases.
+As usual there has been a bunch of Haxelib releases.
```
Comment (skial) [inline src/roundups/325.md]: Good catch, this appears to be the official casing. I'll have to update the title refs as well and try to remember for future roundups.

**Comment by Gama11** [comment]
Haha, guess that means you were expecting this pull? :D

**Comment by skial** [comment]
I would've been surprised if one _didn't_ pop up, my spelling, grammar and punctuation are pretty poor when writing large chunks of text.

Also, I do **appreciate** all the pull requests you have provided, fixing all the things I've over looked. :smile:

**Comment by Gama11** [comment]
Glad to hear that.. btw, "to overlook" is a single word. ;)

**Comment by skial** [comment]
:sob:

**Pull request merged**